interfaces/payment-handler.idl

```diff
@@ -4,83 +4,87 @@
 // See: https://w3c.github.io/payment-handler/
 
 partial interface ServiceWorkerRegistration {
-    [SameObject]
-    readonly attribute PaymentManager paymentManager;
+  [SameObject] readonly attribute PaymentManager paymentManager;
```

Comment:
Changes here confirmed to be all whitespace with `git diff -w --word-diff-regex=.`